README.md

```diff
@@ -135,7 +135,7 @@ var listener = {
   error: (err) => {
     console.error('The Stream gave me an error: ', err);
   },
-  completed: () => {
+  complete: () => {
```

Comment:
Thanks for the typo fixes @Widdershin but could you do these changes instead in the corresponding `markdown/foo` file? Check the first line of this README